modules/rosa-policy-responsibilities.adoc

```diff
@@ -54,3 +59,4 @@ If the `cluster-admin` role is added to a user, see the responsibilities and exc
 |Hardware/AWS global infrastructure |AWS |AWS |AWS |AWS |AWS
 
 |===
+1. If the customer chooses to use their own CNI plugin, the responsibility shifts to both Red Hat and the customer.
```

Comment:
If customer chooses to use their own CNI plugin, the responsibility shifts to Customer. 
OR
Mention Red Hat and Customer in the table with the footnote: If customer chooses to use their own CNI plugin, the responsibility belongs to Customer. If customer uses the default CNI plugin, the responsibility belongs to Red Hat.